examples/computer_vision/unets_tf_keras/model_def.py

```diff
@@ -109,13 +111,15 @@ def build_model(self):
         return model
 
     def build_training_data_loader(self):
-        dataset = tfds.load(
-            'oxford_iiit_pet:3.*.*',
-            split="train",
-            with_info=False,
-            data_dir=self.download_directory,
-            download=not self.data_downloaded,
-        )
+        os.makedirs(self.download_directory, exist_ok=True)
+        with filelock.FileLock(os.path.join(self.download_directory, "download.lock")):
+            dataset = tfds.load(
+                'oxford_iiit_pet:3.*.*',
+                split="train",
+                with_info=False,
+                data_dir=self.download_directory,
+                download=not self.data_downloaded,
```

Comment:
what's this `download` arg for?  I think you can mark it as true or maybe omit it now.  Hopefully `self.data_downloaded` can be done away with too